apps/server/src/setup/index.ts

```diff
@@ -73,7 +73,7 @@ export const srcDir = {
   /** Path to the react app */
   clientDir: isProduction ? join(srcDirectory, 'client/') : join(srcDirectory, '../../client/build/'),
   /** Path to the demo app */
-  demoDir: join(srcDirectory, 'external', config.demo.directory),
+  externalDemoDir: join(srcDirectory, 'external', config.externalDemo),
```

Comment:
suggestion: can we rename this back to `demoDir`? The change doesnt really make too much of a difference, but it would be easier to see what the fix actually is if we removed unnecessary changes